publish_upm.sh

```diff
@@ -1,4 +1,5 @@
-git subtree push --prefix=Assets/Mirror origin upm 
-git fetch
-git tag $1 origin/upm
-git push --tags
+#!/bin/sh
+git subtree split --prefix=Assets/Mirror -b upm 
+git filter-branch --prune-empty --tree-filter 'rm -rf Tests' upm
+git tag $1 upm
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4803501)